sirbot-pyslackers/endpoints/apscheduler.py

```diff
@@ -22,6 +23,23 @@ def create_jobs(scheduler, bot):
 
     scheduler.scheduler.add_job(slack_channel_list, "cron", hour=1, kwargs={"bot": bot})
     scheduler.scheduler.add_job(slack_users_list, "cron", hour=2, kwargs={"bot": bot})
+    scheduler.scheduler.add_job(
+        etc_finance_bell,
+        "cron",
+        day='1-5',
+        hour=11,
+        timezone="America/Eastern",
```

Comment:
Sorry, I think I lied about the zone name - looks like it's `America/New_York`

```bash
$ sudo systemsetup -listtimezones | grep -i east
 Pacific/Easter
$ sudo systemsetup -listtimezones | grep -i york
 America/New_York
```

That's on mac, but [this list](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) seems to confirm